openslides/assignments/models.py

```diff
@@ -249,7 +271,14 @@ def create_poll(self):
         poll = self.polls.create(
             description=self.poll_description_default,
             pollmethod=pollmethod)
-        poll.set_options({'candidate': user} for user in candidates)
+        options = []
+        for user in candidates:
```

Comment:
Ok, now I fetch the related users directly from `AssignmentRelatedUser`. This is now only one query.